packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -63,6 +64,21 @@ export default class PciStoragesContainersAddController {
 
   onTypesFocus() {
     this.displaySelectedType = false;
+    this.loadingPrice = true;
+    this.PciProjectStorageContainersService.getPriceEstimation(
+      this.user.ovhSubsidiary,
+    )
+      .then((price) => {
+        this.price = price;
```

Comment:
```suggestion
        this.price = {
            ...price,
            formatedPrice: this.getFormatedPrice(
                get(price, 'price', 0) / 100000000,
            )
        };
```

Reply:
I get "Shorthand property assignments are valid only in destructuring patterns (74:24)" when I make this change

Reply:
I probably made a syntaxic mistake because this should work :)

Reply:
I tried again with no syntax error and I still get this eslint error.